lib/zipkin-tracer/excon/zipkin-tracer.rb

```diff
@@ -29,14 +29,20 @@ def request_call(datum)
 
     def response_call(datum)
       if span = datum[:span]
-        if status = response_status(datum)
+        status = response_status(datum)
+        if status
           span.record_tag(
             Trace::BinaryAnnotation::STATUS,
             status,
             Trace::BinaryAnnotation::Type::STRING,
             local_endpoint
           )
         end
+        if STATUS_ERROR_REGEXP.match(status.to_s)
+          span.record_tag(Trace::BinaryAnnotation::ERROR, "Error #{status}",
```

Comment:
not terribly important, but Error is already the tag name, so not sure prefixing here clarifies. Ex. I just put the status in this case.